mpas/CMakeLists.txt

```diff
@@ -0,0 +1,129 @@
+cmake_minimum_required(VERSION 3.19)
+
+project(MPAS
+  VERSION 1.0.0
+  LANGUAGES Fortran)
+
+include(${CMAKE_CURRENT_SOURCE_DIR}/MPAS-Model/cmake/Functions/MPAS_Functions.cmake)
+
+list(INSERT CMAKE_MODULE_PATH 0 ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules)
+set(CMAKE_DIRECTORY_LABELS ${PROJECT_NAME})
+include(GNUInstallDirs)
+
+# Build product output locations
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+
+# Set default build type to RelWithDebInfo
+if(NOT CMAKE_BUILD_TYPE)
+  message(STATUS "Setting default build type to Release.  Specify CMAKE_BUILD_TYPE to override.")
+  set(CMAKE_BUILD_TYPE "Release" CACHE STRING "CMake Build type" FORCE)
+  set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
+endif()
+
+# Find C pre-processor
+if(CMAKE_C_COMPILER_ID MATCHES GNU)
+  find_program(CPP_EXECUTABLE NAMES cpp REQUIRED)
+  set(CPP_EXTRA_FLAGS -traditional)
+elseif(CMAKE_C_COMPILER_ID MATCHES "(Apple)?Clang" )
+  find_program(CPP_EXECUTABLE NAMES cpp REQUIRED)
+else()
+  message(STATUS "Unknown compiler: ${CMAKE_C_COMPILER_ID}")
+  set(CPP_EXECUTABLE ${CMAKE_C_COMPILER})
+endif()
+
+# Fortran module output directory for build interface
+set(MPAS_MODULE_DIR ${PROJECT_NAME}/module/${CMAKE_Fortran_COMPILER_ID}/${CMAKE_Fortran_COMPILER_VERSION})
+
+# Install Fortran module directory
+install(DIRECTORY ${CMAKE_BINARY_DIR}/${MPAS_MODULE_DIR}/ DESTINATION ${CMAKE_INSTALL_LIBDIR}/${MPAS_MODULE_DIR}/)
+
+###############################################################################
+# Build MPAS libraries...
+###############################################################################
+
+# MPAS Utilities (Externals)
+add_subdirectory(MPAS-Model/src/external/ezxml)
+add_subdirectory(MPAS-Model/src/external/SMIOL)
```

Comment:
Since you are using SMIOL, why are you looking for PIO in one of the other CMakeLists.txt? SMIOL uses pnetcdf only if I am not mistaken.

Reply:
We are using PIO, so this should be removed here.